pkg/alertmanager/alerts/compat.go

```diff
@@ -0,0 +1,26 @@
+package alerts
+
+// ToProto con
+func ToProto(cfg string, templates map[string]string, user string) (*AlertConfigDesc, error) {
```

Comment:
Who uses it?

Reply:
Not needed in this PR, I'll remove this file